website/docs/advanced/docker/images.md

```diff
@@ -39,9 +39,10 @@ flowchart TD
     svbaseenv --> cnmopsenv[cnmpos-virtual-env]
     svbase & cnmopsenv --> cnmpos[cnmops]
 
-    ubuntu18[Ubuntu 18.04] --> manta[Manta] & melt[MELT] & wham[Wham]
+    ubuntu1804[Ubuntu 18.04] --> manta[Manta] & melt[MELT] & wham[Wham]
```

Comment:
Manta is now on 20.04

Reply:
Good point! Thanks! I updated the figure accordingly.